src/java.base/share/classes/sun/nio/ch/UnixDomainSockets.java

```diff
@@ -44,7 +44,9 @@
 class UnixDomainSockets {
     private UnixDomainSockets() { }
 
-    static final UnixDomainSocketAddress UNNAMED = UnixDomainSocketAddress.of("");
+    private static class UNNAMEDHolder {
```

Comment:
Let's rename this to UnamedHolder to avoid this strange class name.

Reply:
Also renamed.